src/main/java/uk/gov/cshr/domain/Role.java

```diff
@@ -26,6 +27,26 @@ public String getName() {
         return name;
     }
 
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public long getId() {
+        return id;
+    }
+
+    public void setId(long id) {
```

Comment:
Should only have getters and setters for properties we want to get and want to allow to be changed. If we need to build up an object we should use the builder pattern.